### PR TITLE
Include client/app version in message api grpc context metadata

### PR DESCRIPTION
**Pull request by snormore**
Include client/app version in outgoing grpc context metadata

### CODE REVIEW

**Review by neekolas — APPROVED** [review]
Sorry, totally missed this. Looks great.

File: cmd/server/main.go [hunk 1]
```diff
@@ -39,6 +41,13 @@ func main() {
 
 	logger = logging.CreateLogger(opts.LogEncoding, opts.LogLevel)
 
+	shortGitCommit := GitCommit
```
Comment (neekolas) [inline cmd/server/main.go:44]: Would be nicer for readability to break this code into a helper function

File: cmd/server/main.go [hunk 2]
```diff
@@ -39,6 +41,13 @@ func main() {
...
+	if len(shortGitCommit) >= 7 {
+		shortGitCommit = shortGitCommit[:7]
+	}
+	clientVersion := "example-notifications-server-go/" + shortGitCommit
+	appVersion := clientVersion
```
Comment (snormore) [inline cmd/server/main.go:49]: I don't feel great about using the same value for client and app versions, but don't have a better idea until we publish a go client somewhere

Comment (neekolas) [reply cmd/server/main.go:45]: One option is to treat the proto repo version as the go client version. The generated code from the repo is the client.

**Pull request merged**